test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -833,6 +833,60 @@ namespace StackTest {
             $LASTEXITCODE | Should -Be $ExitCodeBadCommandLineParameter
         }
     }
+
+    Context "Startup banner text tests" -Tag Slow {
+        BeforeAll {
+            $outputPath = Join-Path $env:temp "StartupBannerTest-Output-${Pid}.txt"
+
+            $inputPath = Join-Path $env:temp "StartupBannerTest-Input.txt"
+            "exit" > $inputPath
+
+            # Not testing update notification banner text here
+            $origUpdateCheckVal = $env:POWERSHELL_UPDATECHECK
+            $env:POWERSHELL_UPDATECHECK = "Off"
+
+            $spArgs = @{
+                FilePath = $powershell
+                ArgumentList = @("-NoProfile")
+                RedirectStandardInput = $inputPath
+                RedirectStandardOutput = $outputPath
+                WorkingDirectory = $pwd
+                PassThru = $true
+                NoNewWindow = $true
+            }
+        }
+        AfterAll {
+            # Restore original value of $env:POWERSHELL_UPDATECHECK
+            if ($origUpdateCheckVal) {
+                $env:POWERSHELL_UPDATECHECK = $origUpdateCheckVal
+            }
+            else {
+                Remove-Item Env:\POWERSHELL_UPDATECHECK -Force
+            }
+        }
+        BeforeEach {
+            Remove-Item $outputPath -Force -ErrorAction Ignore
```

Comment:
We need to remove `$inputPath`  too. And in AfterAll too.

Reply:
I've add script to remove the stdin file in AfterAll but I do not want to remove it in BeforeEach.  It is created once in BeforeAll and each test uses the same stdin text (`exit`).  There's no point in re-creating that same text in the stdin file for each test.